assets/js/previews/shipping-rates.js

```diff
@@ -5,7 +5,12 @@ import { __ } from '@wordpress/i18n';
 
 export const previewShippingRates = [
 	{
-		destination: {},
+		destination: {
+			country: 'US',
+			state: 'New York',
+			city: 'NYC',
+			postcode: '10007',
+		},
```

Comment:
Not blocking, but I don't think there is a need to add an address to the preview. If my store doesn't sell to the US it would be weird seeing a New York address in the _Cart_ block. I think not showing any address in the editor is ok.